src/sentry/options/defaults.py

```diff
@@ -51,6 +51,13 @@
     flags=FLAG_PRIORITIZE_DISK,
 )
 register("releasefile.cache-limit", type=Int, default=10 * 1024 * 1024, flags=FLAG_PRIORITIZE_DISK)
+register(
+    "releasefile.cache-max-archive-size",
+    type=Int,
+    default=1024 * 1024 * 1024,
```

Comment:
I chose a very high threshold so that this change will only affect the cases that we see failing right now. We might be able to lower this threshold in the future (but *not* caching files has its own drawbacks of course).